contracts/contract_transactions.md

```diff
@@ -79,32 +79,8 @@ init succeeds).
 
 ### Attach Contract Transaction
 
-Attach contract code to an existing account.
-In this case owner == the account address and no new account will be created,
-all other fields are as in create contract.
-
-
-```
-{ owner           :: public_key()
-, nonce           :: pos_integer()
-, code            :: bytes()
-, vm_version      :: integer()
-, abi_version     :: integer()
-, fee             :: amount()
-, deposit         :: amount()
-, amount          :: amount()
-, gas             :: amount()
-, gas_price       :: amount()
-, call_data       :: bytes()
-}
-```
-
-The special variable "caller" will be set to the same value as "owner"
-for the initial call.
-
-The miner will add the return data and the state from the initial call
-to the state tree.
-
+This is introduced in the Fortuna release. Attaching a contract to an account
```

Comment:
Does this mean that a line shall be added in [the point that describes how a contract can be defined](https://github.com/aeternity/protocol/blame/ce23e9d266749acf284a852b167f56c518174cdc/contracts/contract_state_tree.md#L28)?

Reply:
Yes 👍

Reply:
I see addressed in https://github.com/aeternity/protocol/pull/336/commits/44a3c17e6f45d8d1bf580dbca8184ee1cde4f6a6#diff-8d123bd327122eaf06e9e3cb85115851R29